en/docs/chapter_tree/binary_search_tree.md

```diff
@@ -88,25 +88,25 @@ Assuming we choose the smallest node of the right subtree (the next node in in-o
 === "<4>"
     ![bst_remove_case3_step4](binary_search_tree.assets/bst_remove_case3_step4.png)
 
-The operation of removing a node also uses $O(\log n)$ time, where finding the node to be removed requires $O(\log n)$ time, and obtaining the in-order traversal successor node requires $O(\log n)$ time. Example code is as follows:
+Removing a node also uses $O(\log n)$ time, where finding the node to be removed takes $O(\log n)$ time, and obtaining the in-order traversal successor node takes $O(\log n)$ time. Example code is as follows:
 
 ```src
 [file]{binary_search_tree}-[class]{binary_search_tree}-[func]{remove}
 ```
 
 ### In-order traversal is ordered
 
-As shown in the figure below, the in-order traversal of a binary tree follows the "left $\rightarrow$ root $\rightarrow$ right" traversal order, and a binary search tree satisfies the size relationship "left child node $<$ root node $<$ right child node".
+As shown in the figure below, the in-order traversal of a binary tree follows the traversal order of "left $\rightarrow$ root $\rightarrow$ right", and a binary search tree satisfies the size relationship of "left child node $<$ root node $<$ right child node".
 
-This means that in-order traversal in a binary search tree always traverses the next smallest node first, thus deriving an important property: **The in-order traversal sequence of a binary search tree is ascending**.
+This means that when performing in-order traversal in a binary search tree, the next smallest node will always be traversed first, thus leading to an important property: **The sequence of in-order traversal in a binary search tree is ascending**.
 
-Using the ascending property of in-order traversal, obtaining ordered data in a binary search tree requires only $O(n)$ time, without the need for additional sorting operations, which is very efficient.
+Using the ascending property of in-order traversal, obtaining ordered data in a binary search tree takes only $O(n)$ time, without the need for additional sorting operations, which is very efficient.
 
 ![In-order traversal sequence of a binary search tree](binary_search_tree.assets/bst_inorder_traversal.png)
 
 ## Efficiency of binary search trees
 
-Given a set of data, we consider using an array or a binary search tree for storage. Observing the table below, the operations on a binary search tree all have logarithmic time complexity, which is stable and efficient. Only in scenarios of high-frequency addition and low-frequency search and removal, arrays are more efficient than binary search trees.
+Given a set of data, we consider using an array or a binary search tree for storage. Observing the table below, the operations on a binary search tree all have logarithmic time complexity, which is stable and efficient. Only in scenarios of high-frequency addition and low-frequency search and removal, are arrays more efficient than binary search trees.
```

Comment:
optional refinement:
"Given a set of data, we consider using an array or a binary search tree for storage. Observing the table below, the operations on a binary search tree all have logarithmic time complexity, which is stable and efficient. Only in scenarios of high-frequency addition and low-frequency search and removal, are arrays more efficient than binary search trees."
->
"Given a set of data, we consider using an array or a binary search tree for storage. Observing the table below, the operations on a binary search tree all have logarithmic time complexity, which is stable and efficient. Arrays are more efficient than binary search trees only in scenarios involving frequent additions and infrequent searches or deletions."